zed_components/src/zed_camera/src/zed_camera_component.cpp

```diff
@@ -6474,6 +6474,9 @@ void ZedCamera::processOdometry()
 
 void ZedCamera::publishOdom(tf2::Transform & odom2baseTransf, sl::Pose & slPose, rclcpp::Time t)
 {
+  static tf2::Transform lastOdom2baseTransf;
```

Comment:
It's better if you do not use static variables because they can cause issues while using composition.
Move them as members of the class

Reply:
Done